src/org/zaproxy/zap/view/widgets/WritableFileChooser.java

```diff
@@ -24,6 +24,11 @@ public WritableFileChooser(File currentDirectory) {
 	@Override
 	public void approveSelection() {
 		File selectedFile = getSelectedFile();
+		
+		if (selectedFile.length() > selectedFile.getUsableSpace()) {
+			JOptionPane.showMessageDialog(this, "Not enough disk space!");
```

Comment:
The message should be internationalized (as others are in WritableFileChooser).

The change should align with the commit message (or the commit message should be reworded. via interactive rebase to correctly portray the change).